Subject: [PATCH] AudioFX: Fix possible RuntimeException
java.lang.NullPointerException: Attempt to invoke interface 
method 'android.graphics.drawable.Drawable android.view.MenuItem.getIcon()' 
on a null object reference

diff --git a/src/org/cyanogenmod/audiofx/fragment/AudioFxFragment.java b/src/org/cyanogenmod/audiofx/fragment/AudioFxFragment.java
index 846c41b..1ec09da 100644
--- a/src/org/cyanogenmod/audiofx/fragment/AudioFxFragment.java
+++ b/src/org/cyanogenmod/audiofx/fragment/AudioFxFragment.java
@@ -330,7 +330,8 @@
         if (selectedItem != null) {
             selectedItem.setChecked(true);
         }
-        mMenuDevices.setIcon(selectedItem.getIcon());
+        if (selectedItem == null) return;
+        mMenuDevices.setIcon(selectedItem.getIcon());
     }
 
     @Override
